config/SupportedVehicles.xml

```diff
@@ -96,6 +96,30 @@
         <vehicleConfiguration configFileName="FS22_Lizard_Selfmade_Tow_Truck/pickup2017.xml">
             <loadingArea offset="0 0.65 -2.70" width="2.0" height="2.20" length="4.25"/>
             <options noLoadingIfUnfolded="true"/>
+		</vehicleConfiguration>
+        <vehicleConfiguration configFileName="FS22_allPurposeTool/all-purpose-tool.xml">
```

Comment:
Not sure if you meant to include this one as well. I have a version above in the config, but open to changing the loading zone if yours works better. But I did notice yours is above the implement right now:

![image](https://user-images.githubusercontent.com/15822357/165356403-4fb20794-e128-47c7-9d20-64a8652ee40e.png)

Reply:
I didn't notice that you had already done this implementation. Let's keep it, I will remove mine.